_includes/footer.html

```diff
@@ -2,6 +2,7 @@
   <ul>
     <li><a href="/about">About</a></li>
     <li><a href="/learn/municipal-bonds">Learn About Bonds</a></li>
+    <li><a href="https://github.com/h3h/austinprops.city">Feedback</a></li>
```

Comment:
Maybe this should link directly to the Issues page? And people who know WTF Github is can find the code from there if they want?